utils/authRedirect.js

```
@@ -0,0 +1,56 @@
+ *
+ * Custom mobile schemes (`apphemocione:`, `br.com.hemocione.app://`) are valid
+ * destinations, so this cannot simply require http/https.
+ */
+export const resolveAuthRedirect = ({
+  candidate,
+  fallback,
+  currentHostname,
+  token,
+}) => {
+  const safeFallback = parseUrl(fallback || DEFAULT_REDIRECT) ||
+    parseUrl(DEFAULT_REDIRECT);
```

ESLint

[error] Replace `·parseUrl(fallback·||·DEFAULT_REDIRECT)·||⏎···` with `⏎····parseUrl(fallback·||·DEFAULT_REDIRECT)·||`

(prettier/prettier)